src/frontend/src/components/createnewproject/DataExtract.tsx

```diff
@@ -105,7 +105,7 @@ const DataExtract = ({ flag, customDataExtractUpload, setCustomDataExtractUpload
         dispatch(
           CommonActions.SetSnackBar({
             open: true,
-            message: 'Data extract has no features. Please try adjusting your AOI.',
+            message: 'Map features has no features. Please try adjusting your AOI.',
```

Comment:
Could you tweak this to:

'Map has no features. Please try adjusting the map area.'